common/snmp-engine.c

```diff
@@ -976,7 +976,7 @@ snmp_engine_request (const char *hostname, const char *port,
 
 	/* Otherwise flush on the idle callback */
 	else if (!snmp_flush_pending) {
-		server_oneshot (0, request_flush_cb, NULL);
+		server_timer (0, request_flush_cb, NULL);
```

Comment:
Hi Richard,
I think passing 0 here as `period_ms` could trigger the ASSERT in `add_timer` (server_mainloop.c L85).
Even if this timer is not to be recurring, the ASSERT would still demand a non-zero period...

Cheers,
Vlad

Reply:
Thanks, happy with how this is resolved in 30b12f5